packages/near-mobile-wallet/src/lib/init.wallet.ts

```diff
@@ -55,13 +55,13 @@ export const initNearMobileWallet: NearMobileWalletInit = async (config) => {
     async signAndSendTransaction(data) {
       logger.log("[NearMobileWallet]: signAndSendTransaction", data);
 
-      const { contract } = store.getState();
-      if (!contract) {
+      const { contracts } = store.getState();
+      if (contracts.length < 1) {
         throw new Error("Wallet not signed in");
       }
 
       return await nearMobileWallet.signAndSendTransaction({
-        receiverId: contract.contractId,
+        receiverId: contracts[0].contractId,
```

Comment:
Maybe it's best to set the `receiverId` from the `data.receiverId`  (provided in signAndSendTransaction) since now it will be required as mentioned [here](https://github.com/near/wallet-selector/pull/847#:~:text=receiverId%20param%20in%20signAndSendTransaction%20is%20now%20required) OR it needs to be documented that it will default to the first `contractId`

The same applies to other files with similar/same logic.

Reply:
@gtsonevv you can mark this as resolved as I no longer have access on this repo 👍